game-core/build.install4j

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<install4j version="7.0.1" transformSequenceNumber="7">
+<install4j version="7.0.3" transformSequenceNumber="7">
```

Comment:
Probably want to make changes using the same version of the IDE as the program version we install in Travis (install4j will complain in the build log if the XML version is newer than the program version).  In this case, it's pretty obvious there were no unintended (and possibly incompatible) changes made by the IDE, so simply reverting this line should be sufficient.

Reply:
Hmm, interesting that we do not have the latest for install, surprising.
I fixed this in: 9de010076 by downgrading the version value in XML.